packages/langchain-graph-retriever/src/langchain_graph_retriever/adapters/langchain.py

```diff
@@ -13,17 +14,15 @@
     override,
 )
 
+from langchain_graph_retriever._conversion import doc_to_content
 from langchain_graph_retriever.document_transformers.metadata_denormalizer import (
     MetadataDenormalizer,
 )
-from langchain_graph_retriever.types import Edge, IdEdge, MetadataEdge
 
 StoreT = TypeVar("StoreT", bound=VectorStore)
 
-METADATA_EMBEDDING_KEY = "__embedding"
 
-
-class Adapter(Generic[StoreT], abc.ABC):
+class LangchainAdapter(Generic[StoreT], Adapter):
```

Comment:
This `LangChainAdapter` provides the formatting wrappers around things. Most adapters here use it, although the `AstraAdapter` implements `Adapter` directly, which allows it to avoid putting the embedding in the metadata, etc.